tools/aa/BUILD

```diff
@@ -19,11 +19,13 @@ java_binary(
     main_class = "com.google.startupos.tools.aa.AaTool",
     deps = [
         ":config_java_proto",
+        "//common",
+        "//common:dagger_with_annotation_processor",
+        "//common/flags",
```

Comment:
We probably don't need `"//common/flags",` in `AaTool` anymore..

Reply:
@oferb `InitCommand` does use it and it's linked to `aa_tool` as well

Reply:
Ohh right..